### PR TITLE
feat(compose): optimize one function case

### PR DESCRIPTION
**Pull request by julien-f**

### CODE REVIEW

File: src/compose.js [hunk 1]
```diff
@@ -10,11 +10,16 @@
  */
 
 export default function compose(...funcs) {
-  if (funcs.length === 0) {
+  const { length } = funcs
+  if (!length) {
```
Comment (timdorr) [inline src/compose.js]: I'd remove this variable extraction. You don't use it below. It's an unnecessary optimization.

**Comment by timdorr** [comment]
Neat. The branching reads a bit better this way IMHO.

Can you add a test against the new return?

**Comment by julien-f** [comment]
Done.

File: src/compose.js [hunk 2]
```diff
@@ -12,9 +12,13 @@
 export default function compose(...funcs) {
   if (funcs.length === 0) {
```
Comment (dsomel21) [inline src/compose.js]: Should I be using triple = ?

Comment (TrySound) [inline src/compose.js]: Yep. It's strict equality.

**Pull request merged**